semgrep_output_v1.atd

```diff
@@ -1445,8 +1445,10 @@ type scan_configuration = {
     (* Rules sent from the backend. Note that those rules are in JSON
      * form not YAML (which led to some speedup in pysemgrep)
      * TODO? can we type this better *)
-    rules: raw_json; 
+    rules: raw_json;
     inherit triage_ignored;
+    (* From 1.129.0, tells us what merge base to use if it's a diff scan *)
```

Comment:
Is this version right? 1.130.0 is already out, and surely you didn't make the change in Semgrep before making the change here?

Reply:
let me bump that